src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx

```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
 tags:
   - Integrations
   - Amazon integrations
-  - AWS integrations list
+  - Lambda Extension
 ---
+**Work in progress**
 
-The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and setting up your New Relic license key as an environment variable.
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and get insights from your data instently on New Relic One platform.
 
 ## Installation
 
-
 1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
 2. Select **Layers** > **Add Layer**.
 3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
-4. Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
+4a. If you can't find the extension as described in step 4, use the following pattern to get ARN <var>arn:aws:lambda:{your-region}:533243300146:layer:AwsLambdaExtension:{version}</var> The easiest way to get the most recent version would be to start from 1 and save, then push "Edit" button and select the highest version available for the chosen extension.  
```

Comment:
```suggestion
4a. If you can't find the AwsLambdaExtension, use the following pattern to get ARN:
```
 <var>arn:aws:lambda:{your-region}:533243300146:layer:AwsLambdaExtension:{version}</var>
```
To get the most recent version of the extension, start from 1 and save, click **Edit**, and select the highest version available.
```

@4demos What does "start from 1 and save" mean? Do you mean "start from **Step 1**"? Also, is arn all uppercase or lowercase? There are three variations in these steps (arn, Arn, and ARN). I can also make them codified (`arn`).

Reply:
no, start from version 1. arn, Arn and ARN refer to the same entity, codifying would be a good idea

Reply:
overall the versioning is not unified and a version can't be updated, so it's a bit tricky to get the proper (or latest) version in a given region

Reply:
@4demos check out my updated suggestion and lmk what you think